serverless.yml

```diff
@@ -38,6 +41,18 @@ custom:
       Type: String
       Default: ${env:GITHUB_BUILD_COMMENT, self:custom.github-webhook.githubBuildComment}
       Description: For pull request comments, the comment that will initiate a build
+    CbExternalBuildspec:
+      Type: String
+      Default: ${env:CB_EXTERNAL_BUILDSPEC, self:custom.github-webhook.cbExternalBuildspec}
+      Description: If true, CodeBuild will not send "sourceVersion" with request for buildspec (e.g., allowing buildspec to be in an external repo not on same PR number)
+    CbGitRepoEnv:
+      Type: String
+      Default: ${env:CB_GIT_REPO_ENV, self:custom.github-webhook.cbGitRepoEnv}
+      Description: Name of the env used to pass repo clone URL to CodeBuild (optional, if not set, not passed)
```

Comment:
"(Optional) Name of ENV variable that your CodeBuild project uses to know which repo to clone. For use with CB_EXTERNAL_BUILDSPEC."